stix2/properties.py

```diff
@@ -1,5 +1,11 @@
 import re
 import uuid
+from six import text_type
```

Comment:
Can we split these into built-in (standard library) imports, third-party imports (six, dateutil), and internal (`.base`), and alphabetize within each group? I think there are ways to check for this in CI.